src/screens/PoolTogether/PoolTogetherPurchaseConfirm.js

```diff
@@ -174,38 +171,50 @@ class PoolTogetherPurchaseConfirm extends React.Component<Props, State> {
           innerRef={ref => { this.scroll = ref; }}
         >
           <ContentWrapper>
-            <PoolTogetherPurchaseScheme
-              fromValue={tokenValue}
-              fromAssetCode={poolToken}
-              toValue={tokenValue}
-              toAssetCode={t('ticket', { count: tokenValue })}
-              imageSource={poolTogetherLogo}
-            />
-            <ContentRow>
-              <Text label>
-                {t('poolTogetherContent.label.winningChance', {
-                  primaryText: t('percentValue', { value: formatAmount(winChance, 6) }),
-                })}
-              </Text>
-            </ContentRow>
-            <ContentRow style={{ paddingTop: 64 }}>
-              <Text label>{
-                t('label.feeTokenFiat', {
-                  tokenValue: feeDisplayValue,
-                  fiatValue: feeInFiat,
+            <Center>
+              <TokenImage source={{ uri: assetIcon }} />
+              <Spacing h={16} />
+              <BaseText regular>{t('poolTogetherContent.label.youArePurchasing')}</BaseText>
+              <Spacing h={16} />
+              <MediumText giant>
+                {t('poolTogetherContent.label.ticketsReview', {
+                  count: tokenValue,
+                  mediumText: true,
+                  secondary: true,
+                  fontSize: 20,
                 })}
-              </Text>
-            </ContentRow>
-            <ContentRow style={{ paddingTop: 16 }}>
-              <Button
-                title={t('poolTogetherContent.button.purchaseTickets')}
-                onPress={() => {
-                  this.purchasePoolAsset();
-                }}
-                style={{ marginBottom: 13, width: '100%' }}
-                disabled={isDisabled}
-              />
-            </ContentRow>
+              </MediumText>
+              <Spacing h={7} />
+              <BaseText secondary small>{tokenValue} {poolToken}</BaseText>
```

Comment:
Maybe `TokenReviewSummaryComponent` can be reused here?
Also, TokenImage (here and in `TokenReviewSummaryComponent`) misses fallback image, for those tokens that has no icon.

Reply:
Theoretically we can have two separate components. One for just showing the layout (let's say TokenReviewSummaryLayout) and the second for calculations (let's say TokenReviewSummary) that uses TokenReviewSummaryLayout. Here we'd use TokenReviewSummaryLayout too. But I’m afraid the translations magic (`{t('poolTogetherContent.label.ticketsReview', {`) would work with this approach.

Reply:
Or maybe we could just pass in custom amount as a prop (that does not need any formatting)?

Reply:
Custom amount also won't work, maybe if I pass a whole component amount but it kinda defeats the purpose of reusing the component
I mean the "1 ticket" component.